app/components/provider_interface/provider_permissions_list_component.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provider_user.providers.each do |provider| %>
```

Comment:
This will expose the fact that the user has access to a certain provider, even if the `current_provider_user` does not have access to that provider. This problem also exists on the index page by the way.